src/components/ErrorPanel.tsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import cn from 'classnames';
+
+type Props = {
+  errorMessage: string,
+  showError: boolean,
+  clearError: () => void;
+};
+
+export const ErrorPanel: React.FC<Props> = ({
+  errorMessage,
+  showError,
+  clearError,
+}) => {
+  if (!showError) {
+    return null;
+  }
```

Comment:
should you really return null instead of component here? Probably you can just render it conditionally from parent component